acquia/acquia_cms
# ACMS-1272: Added ss package for content model with search templates.

**Motivation**
Fixes #ACMS-1272

**Proposed changes**
Added new site studio pack for content model view template with search.

**Alternatives considered**
NA

**Testing steps**
- Install site with this branch.
- Install ACMS site studio.
- Install ACMS article.
- It should not give any error with ss package import.
- Verify site working fine.

**Merge requirements**
